packages/scene-composer/src/augmentations/components/three-fiber/viewpoint/ViewCursorWidget.spec.tsx

```diff
@@ -2,15 +2,14 @@
 import * as THREE from 'three';
 import React from 'react';
 import renderer from 'react-test-renderer';
+import { act } from '@testing-library/react';
 import { useLoader, useThree } from '@react-three/fiber';
+import { MockTransformControls } from '../../../../../tests/__mocks__/MockTransformControls';
 
-import { ViewCursorWidget } from '../../../../../src/augmentations/components/three-fiber/viewpoint/ViewCursorWidget';
+import { ViewCursorWidget } from '../viewpoint/ViewCursorWidget';
 import { Viewpoint } from '../../../../../src';
```

Comment:
All references to Viewpoint can now be removed. The View Cursor was the only reason it wasn't removed when it's feature was removed.

Reply:
Update coming in next commit. Please sanity check removals.